### ISSUE TITLE
Clarification about checkpoints in training on Imagenet

### ISSUE THREAD

**Issue by sb2nov** [stanford-futuredata/dawn-bench-entries#12]
We're working on a DAWNBench entry that uses a slice of a TPU pod. Each epoch is processed so quickly on the pod that a significant amount of time is now being spent on saving checkpoints. Would it be possible for us to provide a submission where we only checkpoint once at the end of the training run and then run eval to validate accuracy?

As another possibility, we could provide data on two runs, one with checkpointing enabled for every epoch and the other with checkpointing disabled until the end. You could use the timing of the run without checkpointing but inspect the accuracy values along the way via the auxiliary run with checkpointing.

Please let us know if either of these paths would be acceptable.

**Comment by frankchn** [comment]
CC @codyaustun

**Comment by codyaustun** [comment]
@sb2nov quick clarification: do you have to save a checkpoint to validate accuracy? Is running through the validation set also slow?

**Comment by sb2nov** [comment]
Yes. We'll have to switch the infeed queue so ideally it would be great if we can just train for X steps and then run evaluation once.

**Comment by codyaustun** [comment]
Thanks! For this first iteration of DAWNBench, we don't require the time to calculate validation accuracy or create checkpoints. But I think the best approach, in this case, is to make two separate submissions, one with only a checkpoint/validation at the end and one with checkpoints after every epoch. It would be useful to see how this impact performance. Also, in future iterations, we will likely include the time for both validation accuracy and checkpointing

**Comment by sb2nov** [comment]
Thanks @codyaustun we'll do that.
